cmake/InstallLocation.cmake

```diff
@@ -26,3 +26,9 @@ endif()
 if(NOT INSTALL_DOC_DIR)
   set(INSTALL_DOC_DIR ${CMAKE_INSTALL_DOCDIR})
 endif()
+
+set(CMAKE_BUILD_WITH_INSTALL_RPATH OFF)
```

Comment:
Isn't that the default value for this setting anyway?

(I'm phrasing this as a question to be polite, but in fact I'm sure: I double-checked both the documentation and the cmake code, and indeed OFF *is* the default value)

So I would say this line can be removed, it's confusing because it's a no-op.

The rest looks fine to me.

Reply:
That's indeed already the default, I suppose it was just artifacts from my old code i copied from.